egs/chime4/s5_1ch/local/stoi_estoi_sdr.m

```diff
@@ -0,0 +1,48 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Copyright 2017 Johns Hopkins University (Author: Aswin Shanmugam Subramanian)
```

Comment:
I'm not sure what this filename means?  This script at least needs documentation to say what it's about.
Personally I think it's a mistake to rely on matlab as fewer and fewer people are using it, but I won't insist.

Reply:
I added documentation. We are using some third party MATLAB scripts to compute enhancement scores, that is why this script is in MATLAB.